site/src/components/ProxyStatusLatency/ProxyStatusLatency.tsx

```diff
@@ -33,22 +35,29 @@ export const ProxyStatusLatency: FC<ProxyStatusLatencyProps> = ({
   }
 
   if (!latency) {
+    const notAvailableText = "Latency not available";
     return (
-      <Tooltip title="Latency not available">
-        <HelpOutline
-          css={{
-            marginLeft: "auto",
-            fontSize: "14px !important",
-            color,
-          }}
-        />
+      <Tooltip title={notAvailableText}>
+        <>
+          <span css={{ ...visuallyHidden }}>{notAvailableText}</span>
+
+          <HelpOutline
+            css={{
+              marginLeft: "auto",
+              fontSize: "14px !important",
+              color,
+            }}
+          />
+        </>
```

Comment:
Annoyingly, `Tooltip` doesn't accept children arrays, so the `Fragment` is necessary